catalog/spatial/registry/registry-rest-endpoint/src/main/java/org/codice/ddf/registry/rest/endpoint/RegistryRestEndpoint.java

```diff
@@ -55,7 +54,7 @@
 
     private RegistryPublicationService registryPublicationService;
 
-    private RegistryReportMapBuilder reportMapBuilder;
+    private RegistryReportBuilder registryReportBuilder;
 
     public RegistryRestEndpoint() {
         templateLoader = new ClassPathTemplateLoader();
```

Comment:
Is there a reason that the templateLoader is in the RegistryRestEndpoint if we're just passing it in the the RegistryReportBuilder?